src/presentation/controllers/configuration-controller.ts

```diff
@@ -0,0 +1,29 @@
+import { BaseController, GetRequest, RestController } from './base-controller'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Request, Response } from 'express'
+import { ConfigurationRepository } from '../../data-access/repositories/interfaces/configuration-repository'
+import { Exception } from '../../model/exceptions/exception'
+import { Configuration } from '../../model/entities/configuration'
+
+@RestController('/configurations')
+export class ConfigurationController extends BaseController {
+  constructor(
+    private readonly configurationRepository: ConfigurationRepository,
+    private readonly httpErrorHandler: HttpErrorHandler
+  ) {
+    super()
+  }
+
+  @GetRequest()
+  public async getConfigurations(_req: Request, res: Response): Promise<void> {
```

Comment:
We only have one configuration for our system, so I think this should be singular not plural. Especially since you are not returning a list of configurations.

Also, the method is called `getConfiguration`, but you are only returning the Blueprint part of the configuration.
Either the method should be renamed to `getBlueprintConfiguration` or you should return the entire configuration.

Reply:
Has been updated to `getBlueprintConfiguration`. 

I had it in plural, as it is returning the `BlueprintConfiguration` for both `ShowStyle` and `Studio`, thus it being multiple, and in my mind needing to be plural.

Reply:
You are returning one Configuration that has a Studio and Showstyle subsection. It's still just one configuration

Reply:
A good indicator for if something should be plural or not: Is it an array or not?